09_tabular.ipynb

```diff
@@ -9631,8 +9633,20 @@
    "display_name": "Python 3",
    "language": "python",
    "name": "python3"
+  },
```

Comment:
Can you please install nbdev and run `nbdev_install_git_hooks` so you don't get this extra stuff in your PRs? It tends to cause conflicts.